.env

```
@@ -4,15 +4,37 @@ MENDER_SERVER_REPOSITORY=mendersoftware
 MENDER_SERVER_ENTERPRISE_REPOSITORY=mender-server-enterprise
 MENDER_SERVER_TAG=main
 
+MENDER_REGISTRY=registry.mender.io
+MENDER_DOCKER_REGISTRY=docker.io
+MENDER_CLIENT_REPOSITORY=mendersoftware
+
+MENDER_CLIENT_DOCKER_IMAGE=mender-client-docker
+MENDER_CLIENT_DOCKER_TAG=mender-master
+
+MENDER_CLIENT_DOCKER_ADDONS_IMAGE=mender-client-docker-addons
+MENDER_CLIENT_DOCKER_ADDONS_TAG=mender-master
+
+MENDER_CLIENT_QEMU_IMAGE=mender-client-qemu
+MENDER_CLIENT_QEMU_TAG=mender-master
+
+MENDER_CLIENT_QEMU_ROFS_IMAGE=mender-client-qemu-rofs
+MENDER_CLIENT_QEMU_ROFS_TAG=mender-master
+
+MENDER_GATEWAY_QEMU_COMMERCIAL_IMAGE=mender-gateway-qemu-commercial
+MENDER_GATEWAY_QEMU_COMMERCIAL_TAG=mender-master
+
+MENDER_MONITOR_QEMU_COMMERCIAL_IMAGE=mender-monitor-qemu-commercial
+MENDER_MONITOR_QEMU_COMMERCIAL_TAG=mender-master
+
+MENDER_QEMU_ROFS_COMMERCIAL_IMAGE=mender-qemu-rofs-commercial
+MENDER_QEMU_ROFS_COMMERCIAL_TAG=mender-master
```

Once we are renaming,

I would go as far as to say that all of these must have a "VIRTUAL_DEVICE" prefix somehow.

```suggestion
VIRTUAL_DEVICE_DOCKER_IMAGE=mender-client-docker
VIRTUAL_DEVICE_DOCKER_TAG=mender-master

VIRTUAL_DEVICE_DOCKER_ADDONS_IMAGE=mender-client-docker-addons
VIRTUAL_DEVICE_DOCKER_ADDONS_TAG=mender-master

VIRTUAL_DEVICE_QEMU_IMAGE=mender-client-qemu
VIRTUAL_DEVICE_QEMU_TAG=mender-master

VIRTUAL_DEVICE_QEMU_ROFS_IMAGE=mender-client-qemu-rofs
VIRTUAL_DEVICE_QEMU_ROFS_TAG=mender-master

VIRTUAL_DEVICE_QEMU_GATEWAY_IMAGE=mender-gateway-qemu-commercial
VIRTUAL_DEVICE_QEMU_GATEWAY_TAG=mender-master

VIRTUAL_DEVICE_QEMU_MONITOR_IMAGE=mender-monitor-qemu-commercial
VIRTUAL_DEVICE_QEMU_MONITOR_TAG=mender-master

VIRTUAL_DEVICE_QEMU_ROFS_COMMERCIAL_IMAGE=mender-qemu-rofs-commercial
VIRTUAL_DEVICE_QEMU_ROFS_COMMERCIAL_TAG=mender-master
```

Does it sound too crazy? Some reasons to switch:

* The `client` is not really a good keyword: all of the images have the client
* The `commercial` for gateway and monitor are not adding any info, these components are closed source

---

After talking F2F we decided to postpone the renaming to not create confusion between the variables and the docker image names.